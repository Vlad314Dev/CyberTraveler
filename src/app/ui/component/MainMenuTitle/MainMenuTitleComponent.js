import './MainMenuTitleStyle';

import React, { PureComponent } from 'react';

class MainMenuTitleComponent extends PureComponent
{
    render()
    {
        return (
            <div block="MainMenuTitle">
                <div block="MainMenuTitle" elem="General">
                    <span block="MainMenuTitle" elem="Value">Cyber Traveler</span>
                </div>
            </div>
        );
    }
}

export default MainMenuTitleComponent;
