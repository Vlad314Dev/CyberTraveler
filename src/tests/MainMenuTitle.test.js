/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import React from 'react';

import MainMenuTitleComponent from '../app/ui/component/MainMenuTitle/MainMenuTitleComponent';

describe('MainMenuTitle', () => {
    it('Should render', () => {
        // eslint-disable-next-line no-unused-vars
        const component = shallow(
            <MainMenuTitleComponent 
                isLoggedIn={ false }
                nickname={ '' }
            />
        );
    });

    it('Should render layout', () => {
        const component = shallow(
            <MainMenuTitleComponent 
                isLoggedIn={ false }
                nickname={ '' }
            />
        );

        expect(component.getElements()).toMatchSnapshot();
    });
});
