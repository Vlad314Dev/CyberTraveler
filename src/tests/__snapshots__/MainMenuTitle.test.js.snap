// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenuTitle Should render layout 1`] = `
Array [
  <div
    className="MainMenuTitle"
  >
    <div
      className="MainMenuTitle__General"
    >
      <span
        className="MainMenuTitle__Value"
      >
        Cyber Traveler
      </span>
    </div>
  </div>,
]
`;
